components/Order/order_events.rst

```diff
@@ -0,0 +1,23 @@
+Order events
+============
+
+.. note::
+    To have benefit from events you should implement event dispatcher or use `symfony event dispatcher component <http://symfony.com/doc/current/components/event_dispatcher/introduction.html>`_.
```

Comment:
Think `To benefit from events you should implement or use the Symfony's EventDispatcher` would  be better.